### ISSUE TITLE
Update formating for go 1.19 rules

### ISSUE THREAD

**Issue by KristianLyng** [telenornms/skogul#245]
As revealed in test-cases in #240 and as seen in https://go.dev/doc/go1.19 , gofmt has changed. This is a trivial issue, but we need to run gofmt / make fmtfix with Go 1.19.

See https://cloud.drone.io/telenornms/skogul/819/1/3 - this test failure is unrelated to the actual commit being tested, because we just use a genering golang docker-image, which has reasonably been bumped to go 1.19.

I suggest that #240 just incorporates the fix since it's needed to make the checks pass anyway. 

In other words: upgrade to go 1.19, run "make fmtfix", then commit and update the pull request.

**Comment by KristianLyng** [comment]
Largely fixed with #240
